src/file-loader.test.ts

```diff
@@ -0,0 +1,256 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
In order to use `data: expect.any(String),` instead add the entiry data as string. same for ids, it will change on every run.